WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/SummaryTableViewCell.swift

```diff
@@ -7,6 +7,61 @@ struct SummaryTableViewCellPresentation {
     let statusName: String
 }
 
+/// The ViewModel for `SummaryTableViewCell`.
+///
+/// TODO This and that cell class should be renamed to be less ambiguous.
```

Comment:
yea this class sounds like it could be used for any type of summary, but the implementation is pretty much order specific 😅